### PR TITLE
Teams, Sponsors, Participants, Judges, Interests Dashboards to Typescript

### LINKED ISSUE #1612: TypeScript: Interests Dashboard
image

### LINKED ISSUE #1613: TypeScript: Judges Dashboard
image

### LINKED ISSUE #1618: TypeScript: Participants Dashboard
image

### LINKED ISSUE #1619: TypeScript: Sponsors Dashboard
image

### LINKED ISSUE #1621: TypeScript: Teams Dashboard
image

### PR DESCRIPTION
**Pull request by d-leeee**

### CODE REVIEW

File: src/data/admin/Judges.tsx [hunk 1]
```diff
@@ -60,12 +70,14 @@ export const COLUMNS = [
...
 
         const zip = new JSZip();
-        const folder = zip.folder();
+        const folder = zip.folder("photos");
 
-        photos.forEach(({ photo, name }) => {
-          const src = photo.split(",")[1];
-          folder.file(`${name.replace(" ", "_")}.png`, src, { base64: true });
-        });
+        if (folder) {
```
Comment (shahdivyank) [inline src/data/admin/Judges.tsx:81]: what is the if check for?

File: src/data/admin/Teams.tsx [hunk 2]
```diff
@@ -71,10 +84,10 @@ export const COLUMNS = [
     header: "Links",
     meta: { width: "w-2/12" },
     enableSorting: false,
-    cell: ({ getValue }) => (
+    cell: (props: CellContext<Team, Team["links"]>) => (
```
Comment (shahdivyank) [inline src/data/admin/Teams.tsx:87]: these type should also be applied to the other dashboards?

File: src/types/dashboard.ts [hunk 3]
```diff
@@ -5,3 +5,8 @@ export type SearchParams = {
   last: string;
   direction: "prev" | "next";
 };
+
+export type Tags = {
+  text: string;
+  value: number;
```
Comment (shahdivyank) [inline src/types/dashboard.ts]: enum the value

File: src/data/admin/Judges.tsx (hunk 1 above)
Comment (d-leeee) [reply src/data/admin/Judges.tsx:81]: `'folder' is possibly 'null'.`

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**